src/common/utils/model.ts

```diff
@@ -92,7 +92,7 @@ export function fetchAllCountyProjections(snapshotUrl: string | null = null) {
   return cachedCountiesProjections[key];
 }
 
-export function useProjections(location: string, county = null) {
+export function useProjections(location: string, county?: County) {
```

Comment:
This wasn't typed properly before which caused some TypeScript errors from where I was calling it.